roles/openshift_facts/library/openshift_facts.py

```diff
@@ -918,6 +914,51 @@ def get_current_config(facts):
 
     return current_config
 
+def build_kubelet_args(facts):
+    """ Build node kubelet_args """
+    if 'node' in facts:
+        kubelet_args = {}
+        if 'cloudprovider' in facts:
+            if facts['cloudprovider']['kind'] == 'aws':
+                kubelet_args['cloud-provider'] = 'aws'
+                kubelet_args['cloud-config'] = ['/etc/aws/aws.conf']
```

Comment:
Should we actually put this in /etc/aws or should we put it in /etc/origin/cloudprovider/aws.conf ?

Same with the openstack file below...  /etc/origin/cloudprovider/openstack.conf?